cluster/gce/manifests/etcd.manifest

```diff
@@ -45,7 +45,7 @@
         "value": "{{ etcd_protocol }}://{{ hostname }}:{{ server_port }}"
       },
       { "name": "ETCD_SNAPSHOT_COUNT",
-        "value": "10000"
+        "value": "100000"
       }
```

Comment:
do we really need this change? I checked community still using "10000" as value.

Reply:
In that case we should use 10K in arktos run. I noticed recent 1.18 test used 10K. Please check with the team.